spec/lib/apress/utils/extensions/action_view/helpers/form_patch_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+RSpec.describe ActionView::Helpers::FormHelper, type: :helper do
+  describe '#html_options_for_form' do
+    let(:expected_html) do
+      if ::Rails::VERSION::MAJOR < 4.2
+        {
+          'accept-charset' => 'UTF-8',
+          'action' => '/spec'
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.